storyblok/storyblok-design-system
# INT-13 [fix]: Update size title the sbNotification

To solve this problem I added a fixed font size of 24px in the sbNotification title.

## Pull request type

Jira Link: [INT-13: In StoryBlok's Design System, in SbNotification, update size to 24px](https://storyblok.atlassian.net/browse/INT-13)




## How to test this PR
To test this PR, open the design system storybook and click:

Menu > Components > SbNotification

After that, check the font size of the sbNotification title.


## Other information



For any questions I am available.
